static/templates/settings/bot-settings.handlebars

```diff
@@ -50,6 +51,12 @@
                             <label for="create_bot_short_name" generated="true" class="text-error"></label>
                         </div>
                     </div>
+                    <div class="input-group" id="service_base_url_inputbox">
+                        <label for="create_service_base_url">{{t "Outgoing webhook service base URL" }}</label>
+                        <input type="text" name="service_base_url" id="create_service_base_url"
+                            maxlength=100 placeholder="{{t 'http://127.0.0.1:5002/followup' }}" value="" />
```

Comment:
Would it make sense for this example to be `http://127.0.0.1:5002/bot/followup` instead?